keps/sig-node/4580-deprecate-kubelet-runonce/README.md

```diff
@@ -155,7 +155,7 @@ Deprecate and remove kubelet support for RunOnce mode, and mark the `RunOnce` fi
 
 ## Motivation
 
-* RunOnce does not work in systemd mode.
+* RunOnce mode has been broken and is no longer works.
```

Comment:
Wait, it's just completely broken now? Do we know when it broke?

If it's just completely broken, I don't think we need to go through the whole deprecation dance.

Reply:
It seems to have completely stopped working in #90377, because the cpuManager is not initialized in runonce, when calling activePods() will cause nil pointer and kubelet panic. The milestone of #90377 is v1.19....

Reply:
/cc @dchen1107 @mrunalp 
Based on this, do you think we need to go through the whole deprecation dance？

Reply:
in this case we can just delete it straight ahead. 1.19 is reasonably old version to assume that anybody using it would have reported back the issue